tools/wptrunner/wptrunner/wptmanifest/parser.py

```diff
@@ -146,7 +168,10 @@ def line_start_state(self):
             if self.index != self.indent_levels[-1]:
                 raise ParseError(self.filename, self.line_number, "Unexpected indent")
 
-        self.state = self.next_state
+        if self.char() == "#":
```

Comment:
Good catch, we should relax this. The tokenizer now checks this in `line_start_state` with an early out before it checks indentation. Since `comment` tokens can now precede `group_start` ones, the parser has to `consume_comment`s in some more places.